File: src/lib-common/date-get.js
function date_get(x)
{
 if(is_undef(x))
 {
  const n=time_get()
  
  return date_get(n)
 }
 
 check(is_num,x)
 
 const n=mul(x,1000)
 const o=new Date(n)
 const y=o.getFullYear()
 
 let m=o.getMonth()
 
 m=inc(m)
 m=pad_l(m,"0",2)
 
 let d=o.getDate()
 
 d=pad_l(d,"0",2)

 return concat(y,"/",m,"/",d)
}

File: src/lib-common/date-get.cs
fn date_get x
 if is_undef x
  let n call time_get
  
  ret date_get n
 end
 
 check is_num x
 
 let n mul x 1000
 let o new Date n
 let y call o.getFullYear 
 let m call o.getMonth
 let m inc m
 let m pad_l m "0" 2 
 let d call o.getDate 
 let d pad_l d "0" 2

 ret concat y "/" m "/" d
end
